templates/_account/email/email_confirmation_message.txt

```diff
@@ -0,0 +1,8 @@
+{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}Hello from {{ site_name }}!
+
+You're receiving this e-mail because user {{ user_display }} has given yours as an e-mail address to connect their account.
```

Comment:
typo "yours" should be "you"

Reply:
No, this is correct, "has given yours" where the possessive refers to the email address. 

Could be written as "you're receiving this e-mail because {{ user_display }} wants to connect this e-mail address to their account."